c4_set_parameters.m

```diff
@@ -0,0 +1,95 @@
+% set_parameters: sets various parameters for the DLA detection
+% pipeline
+
+%flags for changes
+extrapolate_subdla = 0; %0 = off, 1 = on
+add_proximity_zone = 0;
+integrate          = 1;
+optTag = [num2str(integrate), num2str(extrapolate_subdla), num2str(add_proximity_zone)];
+
+% physical constants
+lya_wavelength = 1215.6701;                   % Lyman alpha transition wavelength  Å
+lyb_wavelength = 1025.7223;                   % Lyman beta  transition wavelength  Å
+lyman_limit    =  911.7633;                   % Lyman limit wavelength             Å
+speed_of_light = 299792458;                   % speed of light                     m s⁻¹
+
+% converts relative velocity in km s^-1 to redshift difference
+kms_to_z = @(kms) (kms * 1000) / speed_of_light;
+
+% utility functions for redshifting
+emitted_wavelengths = ...
+    @(observed_wavelengths, z) (observed_wavelengths / (1 + z));
+
+observed_wavelengths = ...
+    @(emitted_wavelengths,  z) ( emitted_wavelengths * (1 + z));
+
+%release = 'dr12q';
+% download Cooksey's dr7 spectra from this page: 
+% http://www.guavanator.uhh.hawaii.edu/~kcooksey/SDSS/CIV/index.html 
+% go to table: "SDSS spectra of the sightlines surveyed for C IV."
+file_loader = @(mjd, plate, fiber_id) ...
+  (read_spec_dr7(sprintf('c4_data/spectro/1d_26/%04i/1d/spSpec-%05i-%04i-%03i.fit', plate,  mjd,  plate,  fiber_id)));
```

Comment:
Could you make the path of fits files to `data/dr7q/spectra/%04i/1d/spSpec-%05i-%04i-%03i.fit`?
I know this might not be necessary but that would be more consistent with Roman's design.

Reply:
Done!